openreview/venue/process/proposed_assignment_pre_process.js

```diff
@@ -37,4 +40,8 @@ async function process(client, edge, invitation) {
     return Promise.reject(new OpenReviewError({ name: 'Error', message: `Max Papers allowed reached for ${Tools.getPreferredName(profile)}` }));
   }
 
+  if (quota && (submissionEdges.length + 1) > quota && committeeName.includes('Reviewers')) {
+    return Promise.reject(new OpenReviewError({ name: 'Error', message: `You cannot assign more than 3 reviewers to this paper` }));
```

Comment:
make sure you parametrize the number 3 here.

Reply:
remove this condition:  `committeeName.includes('Reviewers')`

the quota is already configured for reviewers only.